pyk/src/pyk/kdist/_kdist.py

```diff
@@ -134,10 +138,13 @@ def _build_target(
         target_id: TargetId,
         args: dict[str, Any],
         *,
+        log_level: int,
         force: bool,
         verbose: bool,
         clean: bool,
     ) -> Path:
+        logging.basicConfig(level=log_level, format=LOG_FORMAT)
```

Comment:
@goodlyrottenapple, could you check please if this solves the logging issue on macOS?